.gitignore

```diff
@@ -13,4 +13,5 @@ htsjdk
 target
```

Comment:
might want to remove the old ant ignores so that people delete them and they don't get confused